lib/main.coffee

```diff
@@ -28,11 +33,37 @@ module.exports =
       lint: (textEditor) =>
         filePath = textEditor.getPath()
         fileText = textEditor.getText()
+        fileDir = _path2.default.dirname(filePath)
+
+        configFile = helpers.findCached fileDir, [
+          '.tidyrc',
+          '.tidyconfig.cfg',
+          '.tidyconfig.txt',
+          'tidyconfig.cfg',
+          'tidyconfig.txt'
+        ]
+        console.info configFile
+        if configFile
+          options = [
```

Comment:
Just initialize `options` once, then use `Array.unshift()` to put the config file setting at the start if found.